modules/videojsVideoProvider.js

```diff
@@ -13,9 +13,6 @@ import { submodule } from '../src/hook.js';
 import stateFactory from '../libraries/video/shared/state.js';
 import { PLAYBACK_MODE } from '../libraries/video/constants/constants.js';
 import { getEventHandler } from '../libraries/video/shared/eventHandler.js';
-/**
- * @typedef {import('../libraries/video/shared/state.js').State} State
```

Comment:
Why revert this? looks like it worked

Reply:
THe duplication check is buggy . I tried to see if reducing the amount of files changed could limit its scope, but that's not the case.